tests/upgrade/test_upgrade_from.sh

```diff
@@ -35,75 +35,67 @@ LOCAL_FORTIO_LOG=/tmp/fortio_local.log
 POD_FORTIO_LOG=/tmp/fortio_pod.log
 
 # Make sure to change templates/*.yaml with the correct address if this changes.
-TEST_NAMESPACE=test
+TEST_NAMESPACE="test"
 
-original_branch=`git rev-parse --abbrev-ref HEAD`
-cur_branch=${original_branch}
-cur_sha=`git rev-parse HEAD`
+original_branch=$(git rev-parse --abbrev-ref HEAD)
 
 installIstioSystemAtVersionHelmTemplate() {
-   helm template ${ISTIO_ROOT}/install/kubernetes/helm/istio \
+   helm template "${ISTIO_ROOT}/install/kubernetes/helm/istio" \
     --name istio --namespace istio-system \
     --set gateways.istio-ingressgateway.replicaCount=4 \
     --set gateways.istio-ingressgateway.autoscaleMin=4 \
-    --set global.hub=${1} \
-    --set global.tag=${2} > ${ISTIO_ROOT}/istio.yaml
+    --set global.hub="${1}" \
+    --set global.tag="${2}" > "${ISTIO_ROOT}/istio.yaml"
 
-   kubectl apply -n istio-system -f ${ISTIO_ROOT}/istio.yaml
+   kubectl apply -n istio-system -f "${ISTIO_ROOT}/istio.yaml"
 }
 
 gitChangeToTag() {
-   original_branch=`git rev-parse --abbrev-ref HEAD`
-   git checkout tags/${1} || die "Cannot git checkout, ensure your workspace is clean."
-   cur_branch=`git rev-parse --abbrev-ref HEAD`
-   cur_sha=`git rev-parse HEAD`
+   original_branch=$(git rev-parse --abbrev-ref HEAD)
```

Comment:
has the original branch been set at line 40?

Reply:
yep, I didn't pay much attention to the logic of this script, I believe this line is not necessary. Want me to remove it?

Reply:
it would be good since you also removed some lines